tox.ini

```diff
@@ -31,7 +31,7 @@ commands =
     lib: pip install --editable .[dev]
     lib: make flake8
     lib: make pylint
-    lib: make test_codebase
+    lib: make test_lib
```

Comment:
I think they're both as vague as each other really. We already had this debate in another PR and settled on ``lib``, I just missed this reference.